spring-boot-project/spring-boot-docs/src/docs/asciidoc/anchor-rewrite.properties

```diff
@@ -466,6 +466,7 @@ production-ready-metrics-export-influx=actuator.metrics.export.influx
 production-ready-metrics-export-jmx=actuator.metrics.export.jmx
 production-ready-metrics-export-kairos=actuator.metrics.export.kairos
 production-ready-metrics-export-newrelic=actuator.metrics.export.newrelic
+production-ready-metrics-export-otlp=actuator.metrics.export.otlp
```

Comment:
This isn't needed. A rewrite is only needed when the ID of an existing section has changed.